### DISCUSSION TITLE
Onvif Client Test tool 22.12 rev 1, getProfiles not detected

### DISCUSSION THREAD

**Discussion by massimoborroni** [onvif/testspecs#42] [Q&A] [answered]
I'm trying to certificate a Client for Profile S, but I fail. Media Streaming 1 test fails because getProfiles tag is not detected. But the wireshark track shows me that it's present. I'm using _ns3__GetProfiles and client retrieves profiles properly. 
Using <ns3:GetProfiles> instead of <trt:GetProfiles> can affect the test? How can I force the tester to properly works with media ver20 and ns3 namespace?
My goal is to pass the client test for profile S.
If you need more information please reply. 

[getProdiles.txt](https://github.com/onvif/testspecs/files/11308118/getProdiles.txt)

**Comment by madhurao68** [comment, marked as answer]
In the attached log GetProfiles is called for media2 service (xmlns:ns3="http://www.onvif.org/ver20/media/wsdl"). But media2 is not a part of Profile S, it is from Profile T and M. To pass Profile S you need to send request using media service: https://www.onvif.org/ver10/media/wsdl/media.wsdl
